src/app/app.component.html

```diff
@@ -8,15 +8,15 @@
 <main class="">
   <div class="">
     <router-outlet name="action"></router-outlet>
-    <div [hidden]="!isExperimentalFeature">
-      <f8-exp-feature-banner>
-      </f8-exp-feature-banner>
-      <div [hidden]="experimentalFeatureEnabled">
-        <f8-exp-feature-page (onExperimentalFeaturesEnabled)="updateFeatureEnabled($event)">
-        </f8-exp-feature-page>
+    <div [hidden]="!isExperimentalFeature"> <!--if not exp feature, hide-->
+      <feature-banner level="{{featureEnablementLevel}}">
```

Comment:
Curious why the f8 prefix was dropped?  I believe platform is still following this convention?

https://angular.io/guide/styleguide#style-02-07

Reply:
doing a general seach on`selector: 'f8-`in our project I end-up with `f8-app`, `f8-error`, `f8-workspaces-notifications` not sure it's wildly followed.

Reply:
@joshuawilson - we should be enforcing f8 prefix use in the linter.  Removal of these will lead to namespace conflicts with components.

Reply:
I agree. I'll work on the linter or an issue for this. meanwhile we should be using a namespace prefix of `f8-`